app/models/category.rb

```diff
@@ -1118,6 +1118,46 @@ def has_restricted_tags?
     tags.count > 0 || tag_groups.count > 0
   end
 
+  def self.reorder_categories!
+    categories =
+      if SiteSetting.fixed_category_positions
```

Comment:
Do we need this when the method is called when `!SiteSetting.fixed_category_positions`?